### DISCUSSION TITLE
How to get active tenant in the migration handle method?

### DISCUSSION THREAD

**Discussion by sasokovacic** [hyn/multi-tenant#999] [Q&A]
I need to update some fields after migration execution. I need to be able to access active tenant inside this migration.
Is it possible?

**Comment by bkintanar** [comment]
I suggest to run a command after running the migration.

**Reply by sasokovacic** [reply]
Do you mean to run the artisan command inside the migration file or to run it manually? The latter is not acceptable. I have to do it one time only and automatically since I'm deploying applications with ansible.
